### PR TITLE
[Frontend] - Product Gallery Component 

### PR DESCRIPTION
**Pull request by TreshAnn**
## Ticket/s

- [Product Gallery Component](https://trello.com/c/WkzgFy11)

## Changes

- Add ProductGallery Component

## Screenshot
- Website View:
image
- Mobile View

image

## How to test?

-  run **pnpm  run dev**
- After compilation is complete
- Insert to url http://localhost:5173/

### CODE REVIEW

**Review by sandrex-SGV — CHANGES_REQUESTED** [review]
Component does what it needs to do but I suggest to change the image holder's height when the screen is smaller (mobile) because the product image gets stretched. Thanks!

**Comment by TreshAnn** [comment]
> Component does what it needs to do but I suggest to change the image holder's height when the screen is smaller (mobile) because the product image gets stretched. Thanks!

Done, Sir @5andrex , please check. thanks!

**Review by sandrex-SGV — APPROVED** [review]

File: packages/ui/package.json [hunk 1]
```diff
@@ -16,14 +16,16 @@
...
     "@emotion/styled": "^11.11.0",
     "@hookform/resolvers": "^3.1.1",
+    "@mantine/carousel": "^6.0.19",
     "@mantine/core": "^6.0.17",
     "@mantine/hooks": "^6.0.17",
+    "@types/react": "^18.2.0",
+    "@types/react-dom": "^18.2.0",
     "@types/styled-components": "^5.1.26",
     "clsx": "^2.0.0",
+    "embla-carousel-react": "8.0.0-rc11",
```
Comment (dandalandone) [inline packages/ui/package.json]: remove if unused

**Review by dandalandone — APPROVED** [review]
LGTM, but please address the unused Library

File: packages/ui/package.json (hunk 1 above)
Comment (TreshAnn) [reply packages/ui/package.json]: done, sir @dandalandone ! thanks

**Pull request merged**